packages/merge-on-green/src/merge-logic.ts

```diff
@@ -52,6 +52,7 @@ interface RequiredChecksByLanguage {
 interface RepoOverrides {
   repo: string;
   requiredStatusChecks: string[];
+  useBranchProtectionRules: boolean;
```

Comment:
I would make this optional, since it might not be populated:

`useBranchProtectionRules?: boolean`.